content/usecaseItems.md

```diff
@@ -0,0 +1,31 @@
+---
+case_0_id: 0
+case_0_panel:
+  # - title: 
+  #  tooltip: Home
+  #  route: /usecase_0  
+  - title: Conceptual overview of the use case
```

Comment:
I think this would have to be a name that's descriptive for the entire usecase, not just the first panel. Maybe "Example use case" is a better fit?

Reply:
Thanks for the comment. I rename it as "example_usecase".